invenio_theme_tugraz/webpack.py

```diff
@@ -17,9 +17,10 @@
             entry={
                 "invenio-theme-tugraz-theme": "./less/invenio_theme_tugraz/theme.less",
                 "invenio-theme-tugraz-js": "./js/invenio_theme_tugraz/theme.js",
+                "invenio-theme-tugraz-unlock": "./js/invenio_theme_tugraz/unlock.js",
             },
             dependencies={
-                # add any additional npm dependencies here...
+                "jquery": "~3.2.1",  # zammad-form, semantic-ui's modals
```

Comment:
did you checked which jquery version InvenioRDM is using?

Reply:
I did
at the time I wrote the code, this version was the correct one
now, `invenio_previewer` has moved on to `"^3.3.1"`, whereas other invenio-modules are still at `"^3.2.1"`

changed to `"^3.2.1"`, which should fit with current invenio